### DISCUSSION TITLE
Making the vega-lite example galery polyglot ?

### DISCUSSION THREAD

**Discussion by behrica** [vega/altair#2511] [General]
I started here
https://github.com/vega/vega-lite/issues/7758

a discussion to make the official vega lite example gallery polyglot,
so showing the specification of all the  example in various languages.

Maybe you are interested.

**Comment by behrica** [comment]
To bring this forward, I should create an vega lite example in various languages.
Could somebody here help me with this ?

So converting:

```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
  "description": "A simple bar chart with embedded data.",
  "data": {
    "values": [
      {"a": "A", "b": 28}, {"a": "B", "b": 55}, {"a": "C", "b": 43},
      {"a": "D", "b": 91}, {"a": "E", "b": 81}, {"a": "F", "b": 53},
      {"a": "G", "b": 19}, {"a": "H", "b": 87}, {"a": "I", "b": 52}
    ]
  },
  "mark": "bar",
  "encoding": {
    "x": {"field": "a", "type": "nominal", "axis": {"labelAngle": 0}},
    "y": {"field": "b", "type": "quantitative"}
  }
}
``` 

into the altair python code ?
Thanks

**Reply by jakevdp** [reply]
I think this is the same example? https://altair-viz.github.io/gallery/simple_bar_chart.html

**Comment by behrica** [comment]
yes, thanks. I did not "search" in you examples.

**Comment by behrica** [comment]
Just for information .
Do you maintain a "complete" example gallery ?

So did you recreate an Altair example for all examples here: http://vega.github.io/vega-lite/examples/

It looks like being the case.

**Reply by jakevdp** [reply]
Many of the Vega-Lite examples are recreated in Altair's gallery, but certainly not all. Altair is still on Vega-Lite 4.8, so there are several chart types in the gallery that cannot be created by Altair - namely anything with an arc mark.

**Comment by behrica** [comment]
Ok, would you eventually consider to move your examples to the vega-lite website,
in case they go ahead with https://github.com/vega/vega-lite/issues/7758 ?

You should be able to just "move" them in one go from here
https://github.com/altair-viz/altair/tree/master/altair/examples

They might need some bulk renaming, as the vega lite website uses "name" while your site uses "title".
Both are in this file:
https://github.com/vega/vega-lite/blob/master/site/_data/examples.json

**Reply by jakevdp** [reply]
If by move the examples, you mean remove them from the altair repository and only keep them in the vega-lite repository, no we probably will not be open to that. For one thing, the examples within this repo are an important part of our integration test suite, and when Altair is upgraded they frequently need to be updated for new syntax. Trying to do that across multiple repositories would add unnecessary complication.
